package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@rdkit/rdkit",
+  "version": "2021.3.1-beta.14",
```

Since this is going to be automatically set during the build process anyway, I would prefer to put a placeholder here instead of an actual version number.
Perhaps something like:
```suggestion
  "version": "PLACEHOLDER",
```
I tried this out locally and it doesn't seem to cause problems for npm